sdk/samples/core/mosquitto_multiple_rpc_sample.c

```diff
@@ -35,7 +35,7 @@ static const az_span stop_module_command_name = AZ_SPAN_LITERAL_FROM_STR("stopMo
 static const az_span server_subscription_topic_format
     = AZ_SPAN_LITERAL_FROM_STR("device/{executorId}/command/{name}");
 static const az_span client_subscription_topic_format
-    = AZ_SPAN_LITERAL_FROM_STR("device/{executorId}/command/{name}/__for_{invokerId}");
+    = AZ_SPAN_LITERAL_FROM_STR("device/{executorId}/command/{name}/for/{invokerClientId}");
```

Comment:
Can't we use the defaults and avoid creating special topics for the samples? (i.e. remove all `*_topic_*` definitions, simplifying the code a bit).